poetry.lock

```diff
@@ -1,18 +1,28 @@
+# This file is automatically @generated by Poetry 1.4.0 and should not be changed by hand.
+
 [[package]]
 name = "anyascii"
 version = "0.3.1"
 description = "Unicode to ASCII transliteration"
 category = "main"
 optional = false
 python-versions = ">=3.3"
+files = [
+    {file = "anyascii-0.3.1-py3-none-any.whl", hash = "sha256:8707d3185017435933360462a65e2c70a4818490745804f38a5ca55e59eb56a0"},
+    {file = "anyascii-0.3.1.tar.gz", hash = "sha256:dedf57728206e286c91eed7c759505a5e45c8cd01367dd40c2f7248bb15c11f6"},
+]
```

Comment:
Hmm, this metadata will be super hard to review since it looks like Poetry has changed their file format :/ I guess I’ll just trust the process.